### PR TITLE
Update README.md and Release Cheat Sheet

### PR DESCRIPTION
**Pull request by nikhil-thomas**
Update README.md with links to docs and example for v0.13.0 release

Update Triggers Release Cheat Sheet

Signed-off-by: Nikhil Thomas <nikthoma@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [X] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: tekton/release-cheat-sheet.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ the triggers repo, a terminal window and a text editor.
 
 1. `cd` to root of Triggers git checkout.
 
+1. 1. Make sure the release `Task` and `Pipeline` are up-to-date on the
+      cluster.
+
+   - [publish-operator-release](https://github.com/tektoncd/triggers/blob/main/tekton/publish.yaml)
```
Comment (dibyom) [inline tekton/release-cheat-sheet.md:14]:
```suggestion
   - [publish-triggers-release](https://github.com/tektoncd/triggers/blob/main/tekton/publish.yaml)
```

File: tekton/release-cheat-sheet.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ the triggers repo, a terminal window and a text editor.
...
+
+     This task uses [ko](https://github.com/google/ko) to build all container images we release and generate the `release.yaml`
+     ```shell script
+     kubectl apply -f tekton/publish.yaml
+     ```
+   - [operator-release](https://github.com/tektoncd/triggers/blob/main/tekton/release-pipeline.yaml)
```
Comment (dibyom) [inline tekton/release-cheat-sheet.md:20]:
```suggestion
   - [triggers-release](https://github.com/tektoncd/triggers/blob/main/tekton/release-pipeline.yaml)
```

**Review by dibyom — APPROVED** [review]

**Comment by jmcshane** [comment]
@nikhil-thomas could you address the two comments left by @dibyom so we can get this documentation merged this week?

**Comment by nikhil-thomas** [comment]
> @nikhil-thomas could you address the two comments left by @dibyom so we can get this documentation merged this week?

oops, i was away for 2 weeks.

Thank you @dibyom 

does this pr need any additional updates.

**Comment by dibyom** [comment]
I think we should be all set here!

**Pull request merged**